tests/test_diffusers.py

```diff
@@ -2421,6 +2422,144 @@ def test_stable_diffusion_pix2pix_euler(self):
         self.assertLess(np.abs(image_slice.flatten() - expected_slice).max(), 1e-3)
 
 
+class GaudiStableDiffusionImg2ImgPipelineTests(TestCase):
+    """
+    Tests the class StableDiffusionImg2ImgPipeline for Gaudi.
+    Adapted from: https://github.com/huggingface/diffusers/blob/v0.26.3/tests/pipelines/stable_diffusion/test_stable_diffusion_img2img.py
```

Comment:
Can you check if Diffusers 0.29.2 brings any changes?

Reply:
No changes